docs/snippets/common/storybook-preview-register-msw-addon.js.mdx

```diff
@@ -10,16 +10,18 @@ import { initialize, mswDecorator } from 'msw-storybook-addon';
  */
 initialize();
 
-export const parameters = {
-  actions: { argTypesRegex: '^on[A-Z].*' },
-  controls: {
-    matchers: {
-      color: /(background|color)$/i,
-      date: /Date$/,
+export default {
+  parameters: {
+    actions: { argTypesRegex: '^on[A-Z].*' },
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/,
+      },
     },
   },
 };
 
 // Provide the MSW addon decorator globally
-export const decorators = [mswDecorator];
+export default { decorators: [mswDecorator] };
```

Comment:
@shilman when you're able can you move this decorator to the export default above so that it aligns it the rest of the snippets that mention them to avoid misleading folks? Please feel free to correct me if I'm wrong on this.